stdlib/ballerina-websub/src/main/ballerina/websub/hub_service.bal

```diff
@@ -246,7 +246,8 @@ function verifyIntent(string callback, string topic, map params) {
     int leaseSeconds;
 
     if (params.hasKey(HUB_LEASE_SECONDS)) {
-        match (<int>params[HUB_LEASE_SECONDS]) {
+        string strLeaseSeconds = <string>params[HUB_LEASE_SECONDS];
```

Comment:
Type of a value in an unconstrained map is `any`. Therefore, since this is an any-to-string conversion, shouldn't this be an unsafe-conversion?

Btw, why do we need to use a generic map here? Can this map contains values of different types?

Reply:
The params map could be either form params or query params. Since both return a constrained map now, params can be declared a constrained map - fixed with f528ef0